### PR TITLE
Updated Mobile Structure

### PR DESCRIPTION
**Pull request by jdiehl**
* Re-added deploying-native-app
* Reorganized some content pages
* Added better summary pages

### CODE REVIEW

**Comment by jdiehl** [comment]
@ConnorLand I have outlined the content for the mobile start page (/refguide/mobile/). Can you take a look and work some editorial magic?

**Review by ConnorLand — APPROVED** [review]

**Comment by ConnorLand** [comment]
> @ConnorLand I have outlined the content for the mobile start page (/refguide/mobile/). Can you take a look and work some editorial magic?

It would be my pleasure. I will add my edits in this PR and then merge it.

**Review by ConnorLand — CHANGES_REQUESTED** [review]
I'll make some edits here and then merge into the working branch.

File: content/en/docs/refguide/mobile/_index.md [hunk 1]
```diff
@@ -17,35 +17,39 @@ Mendix allows you to quickly build true [native mobile apps](/refguide/native-mo
...
-Progressive web apps have three main characteristics:
+## 4 Designing Mobile User Interfaces
 
-* **Installable —** PWAs let you add your app to your user’s home screen and start a full screen app. This makes PWAs feel more fully-capable native apps.
-* **Reliable —** Using service workers, PWAs can work offline or partially offline. Mendix PWAs can work partially offline (resources like styling, pages, and images are cached) or fully offline (like native mobile apps).
-* **Capable —** PWAs can leverage several device capabilities like the camera and location, and can offer support for web push notifications. Note that support for features depend on which browser is used.
+Well-designed mobile apps take various design requirements into consideration. For example, users frequently change location while using them, often do so on a small screen, and might only use the app in short bursts of activity. Luckily Mendix uses technology that addresses these requirements, and provides design documentation to help you build user-friendly apps.
 
-## 4 Offline-First Apps
+* Native Apps are styled using React Native
```
Comment (ConnorLand) [inline content/en/docs/refguide/mobile/_index.md]: Comment for reference.

**Review by ConnorLand — APPROVED** [review]

**Pull request merged**